docs/api/migration-guides/v2-primitives.mdx

```diff
@@ -0,0 +1,1199 @@
+---
+title: Migrate to the V2 primitives
+description: Migrate from using the original primitives (Estimator and Sampler) to using the updated V2 primitives (EstimatorV2 and SamplerV2)
+
+---
+
+<span id="v2-primitives"></span>
+# Migrate to the Qiskit Runtime V2 primitives
+
+This guide describes what changed in the Qiskit Runtime V2 primitives and why, describes each new primitive in detail, and gives examples to help you migrate code from using the legacy primitives to the V2 primitives.  The examples in the guide all use the Qiskit Runtime primitives, but in general, the same changes apply to the other primitive implementations.  The functions unique to Qiskit Runtime such as error mitigation remain unique to Qiskit Runtime. For information about the changes to the Qiskit reference primitives (now called *statevector* primitives), see (the qiskit.primitives section)[qiskit-1.0-features#qiskit.primitives] in the Qiskit 1.0 feature changes page. 
+
+## Overview
+
+Version 2 of the primitives is introduced with a new base class for both Sampler and Estimator ([BaseSamplerV2](/api/qiskit/qiskit.primitives.BaseSamplerV2) and [BaseEstimatorV2](/api/qiskit/qiskit.primitives.BaseEstimatorV2)), along with new types for their inputs and outputs. The emphasis of this new version is on performing vectorized calls to the primitive `.run()` methods so that you can efficiently specify sweeps over parameter value sets and observables. See [StatevectorSampler](/api/qiskit/qiskit.primitives.StatevectorSampler) and [StatevectorEstimator](/api/qiskit/qiskit.primitives.StatevectorEstimator) for V2 primitive reference implementations.
```

Comment:
This line 
> The emphasis of this new version is on performing vectorized calls to the primitive .run() methods so that you can efficiently specify sweeps over parameter value sets and observables

talks about the same thing as the first line below - i.e. 

> A new interface that lets you specify a single circuit and multiple observables (if using Estimator) and parameter value sets for that circuit. Previously, you had to specify the same circuit multiple times to match the size of the data to be combined.

We can probably merge them somehow, so it's not redundant.

Reply:
In addition, this part

> See [StatevectorSampler](https://qiskit-docs-preview-pr-711.1799mxdls7qz.us-south.codeengine.appdomain.cloud/api/qiskit/qiskit.primitives.StatevectorSampler) and [StatevectorEstimator](https://qiskit-docs-preview-pr-711.1799mxdls7qz.us-south.codeengine.appdomain.cloud/api/qiskit/qiskit.primitives.StatevectorEstimator) for V2 primitive reference implementations.

is better suited to be under `Migrate to the Qiskit Runtime V2 primitives`, where it talks about the reference implementation.